test/ui/VIewRolesUITests.ts

```diff
@@ -0,0 +1,34 @@
+import { describe } from 'mocha'
+import { BuildDriver } from './Driver'
+import { By, type ThenableWebDriver } from 'selenium-webdriver'
+import { expect } from 'chai'
+
+describe('View Roles test', async () => {
+    let driver: ThenableWebDriver
+
+    // const options = new Options()
+    // const headless =
+    // process.env.RUN_HEADLESS != null ? 'headless' : 'disable-gpu'
+    // options.addArguments('no-sandbox', headless)
+
+    beforeEach(async () => {
+        driver = BuildDriver()
+    })
+
+    afterEach(async () => {
+        await driver.quit()
+    })
+
+    it('View and Navigate on View Roles Page', async () => {
+        // @ts-expect-error/avoid-error-as-this-will-work
```

Comment:
Is this needed (it was needed in my example, but might not be needed here)

Reply:
yes it throughs an error without this unfortunately